RecipeRoulette-inc/RecipeRouletteV.2
# Aa/allergy search

feature: allergy search

testing instructions:
login to a user with allergies, and see how the searched recipes don't have those allergens.
